server/models/Document.js

```diff
@@ -100,6 +102,20 @@ const Document = sequelize.define(
         const slugifiedTitle = slugify(this.title);
         return `/doc/${slugifiedTitle}-${this.urlId}`;
       },
+      tasks: function () {
+        if (!this.text) {
+          return [];
+        }
+
+        let countCheckbox = (this.text.match(CHECKBOX_REGEX) || []).length;
+        if (!countCheckbox) {
+          return [];
+        }
+
+        let countTicked = (this.text.match(TICKED_CHECKBOX_REGEX) || []).length;
+
+        return [countTicked, countCheckbox];
+      },
```

Comment:
```js
tasks: {
  completed: 1,
  total: 4
}
```

Thoughts on this interface?

Reply:
Sure! It is more descriptive and maintainble.